### DISCUSSION TITLE
캐릭터 회전

### DISCUSSION THREAD

**Discussion by ghost** [naverz/zepeto-studio-kor#536] [5. Q&A]
움직일때 회전값이 변경되는건 못바꾸나요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 컨트롤러로 이동을 해도 캐릭터가 계속 한 방향을 바라보기를 원하시는 걸까요?

**Reply by ghost** [reply]
네

**Reply by Zepeto-TechEvangelist** [reply]
기본적으로는 지원하지 않습니다.
캐릭터 움직임에 대한 커스터마이징을 원한다면 캐릭터 컨트롤러를 사용하지 않고 
직접 캐릭터를 제어하도록 기능 개발을 해야 합니다.
